src/ex1/scripts/domScripts.js

```diff
@@ -0,0 +1,195 @@
+const taskInput = document.querySelector(".task-input");
+const addTaskButton = document.querySelector(".task-add-btn");
+const tasksList = document.querySelector(".tasks-list");
+const filterOptiopn = document.querySelector(".filter-tasks");
+const searchInput = document.querySelector(".search-input");
+
+addTaskButton.addEventListener("click", onAddTaskClick);
+tasksList.addEventListener("click", onTaskButtonsClick);
+filterOptiopn.addEventListener("click", onFilterOptionChange);
+searchInput.addEventListener("keyup", onSearchInputKeyUp);
+
+taskInput.addEventListener("keypress", function (event) {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    addTaskButton.click();
+  }
+});
+
+function onAddTaskClick(e) {
+  const isOk = canProceed();
+  if (isOk) {
+    const taskDiv = document.createElement("div");
+    taskDiv.classList.add("task");
+
+    const gripLines = createGridLines();
+    taskDiv.appendChild(gripLines);
+
+    const taskContent = createTaskContent();
+    taskDiv.appendChild(taskContent);
+
+    const completeTaskButton = createCompleteBtn();
+    taskDiv.appendChild(completeTaskButton);
+
+    const removeTaskButton = createRemoveBtn();
+    taskDiv.appendChild(removeTaskButton);
+
+    addTaskDivEventListeners(taskDiv);
+    tasksList.appendChild(taskDiv);
+    saveTaskToLocalStorage(taskContent.innerText, false);
+    toggleEmptyMsg();
+    taskInput.value = "";
+  }
+}
+
+function canProceed() {
+  const isAlertOn = isAlertShown();
+  if (taskInput.value === "") {
+    if (!isAlertOn) {
+      toggleAlert();
+    }
+    return false;
+  } else {
+    if (isAlertOn) {
+      toggleAlert();
+    }
+    return true;
+  }
+}
+
+function createGridLines() {
+  const gripLines = document.createElement("div");
+  gripLines.classList.add("grip-lines-icon");
+  gripLines.innerHTML = '<i class="fas fa-grip-lines"></i>';
+  return gripLines;
+}
+
+function createTaskContent(content = null) {
+  const newTask = document.createElement("p");
+  newTask.classList.add("task-item");
+  if (content) {
+    newTask.innerText = content;
+  } else {
+    newTask.innerText = taskInput.value;
+  }
+  return newTask;
+}
+
+function createCompleteBtn(isCompleted = false) {
+  const completeTaskButton = document.createElement("button");
+  completeTaskButton.classList.add("task-complete-btn");
+  completeTaskButton.classList.add("hide");
```

Comment:
you can add multiple classes using comma `classList.add("task-complete-btn", "hide");`